src/PrestaShopBundle/Controller/Admin/CombinationController.php

```diff
@@ -23,45 +23,38 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-
-
 namespace PrestaShopBundle\Controller\Admin;
 
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use PrestaShopBundle\Form\Admin\Product\ProductCombination;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use PrestaShopBundle\Model\Product\AdminModelAdapter as ProductAdminModelAdapter;
 
-class CombinationController extends Controller
+class CombinationController extends FrameworkBundleAdminController
 {
     public function generateCombinationFormAction($combinationIds)
     {
         $response = new Response();
-        $result = '';
 
-        $combinations = explode('-', $combinationIds);
-        if ($combinationIds == 0 || count($combinations) == 0) {
+        $combinationIds = explode('-', $combinationIds);
+        if ($combinationIds == 0 || count($combinationIds) == 0) {
```

Comment:
I guess `$combinationIds == 0` can be replaced by `$combinationIds === false`.

Reply:
@Quetzacoalt91 well, most of the time PrestaShop needs lot more than 200 max_input_vars, in 1.7 it's not a big issue but i remember that in 1.6 we needed it at over 5000 in some scenarios, i think that people used to set this param to higher value

Reply:
Yeah, we probably both remember the error on the translation page. :)
And we may actually reduce the current value, depending on the server configuration.

If the merchants are used to it, then this should become a prerequisite of the installation. In the meantime, we should first check the current value and warn the merchant about the temporary increased value.